### PR TITLE
Revise descriptions, add tags, cryptoprocessor-management to desktop-…

### PR DESCRIPTION
**Pull request by mvincerx**
…kde-games.

Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: bundles/ddd [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: ddd
-# [DESCRIPTION]: graphical debugger front-end; GDB, DBX, Ladebug, JDB, Perl, Python
+# [DESCRIPTION]: Graphical debugger front-end; GDB, DBX, Ladebug, JDB, Perl, Python.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Data Science, Developer Tools, Python, perl
```
Comment (bryteise) [inline bundles/ddd]: Not data science.

File: bundles/database-basic [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: database-basic
-# [DESCRIPTION]: Run a SQL database
+# [DESCRIPTION]: Run an SQL database.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Data Science, Tools and Utilities, database
```
Comment (bryteise) [inline bundles/database-basic]: not data science.

File: bundles/database-basic-dev [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # [DESCRIPTION]: All packages required to build the database-basic bundle.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Data Science, Tools and Utilities, Developer Tools, database
```
Comment (bryteise) [inline bundles/database-basic-dev]: not data science.

File: bundles/desktop-apps [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: desktop-apps
-# [DESCRIPTION]: Applications for the desktop
+# [DESCRIPTION]: Applications for the desktop.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Other, Productivity, Developer Tools, application
```
Comment (bryteise) [inline bundles/desktop-apps]: I'd drop developer tools.

File: bundles/desktop-assets [hunk 5]
```diff
@@ -2,7 +2,7 @@
 # [DESCRIPTION]: Images and Icons for the desktop
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Other, application
```
Comment (bryteise) [inline bundles/desktop-assets]: I think this should drop application.

File: bundles/desktop-gnomelibs [hunk 6]
```diff
@@ -1,5 +1,5 @@
 # [TITLE]: desktop-gnomelibs
-# [DESCRIPTION]: Helper bundle with common libraries used by desktopy things
+# [DESCRIPTION]: Helper bundle with common libraries and desktop utilities.
 # [STATUS]: Active
 # [CAPABILITIES]:
 # [TAGS]:
```
Comment (bryteise) [inline bundles/desktop-gnomelibs]: tag could be other for this one.

File: bundles/desktop-kde-apps [hunk 7]
```diff
@@ -1,8 +1,8 @@
 # [TITLE]: desktop-kde-apps
-# [DESCRIPTION]: Applications for the desktop from the KDE ecosystem
+# [DESCRIPTION]: Applications for the desktop from the KDE ecosystem.
 # [STATUS]: Active
 # [CAPABILITIES]:
-# [TAGS]:
+# [TAGS]: Other, Productivity, Developer Tools, application
```
Comment (bryteise) [inline bundles/desktop-kde-apps]: Drop developer tools I think.

**Comment by bryteise** [comment]
This needs a few fixes.

**Comment by mvincerx** [comment]
See latest push. See also my email. Thanks.

**Comment by bryteise** [comment]
Merged.

**Pull request closed without merging**